src/StelMainView.cpp

```diff
@@ -929,8 +929,14 @@ void StelMainView::init()
 	stelApp->init(configuration);
 	//this makes sure the app knows how large the window is
 	connect(stelScene,SIGNAL(sceneRectChanged(QRectF)),stelApp,SLOT(glWindowHasBeenResized(QRectF)));
+	QObject::connect(stelScene, &StelGraphicsScene::sceneRectChanged, [&](const QRectF& rect)
+	{
+        stelApp->glPhysicalWindowHasBeenResized(getPhysicalSize(rect));
```

Comment:
Let me know if this lambda is acceptable

Reply:
Why not use ifdefs for spout within glWindowHasBeenResized()?

Reply:
Because that class, that has glWindowHasBeenResized() has no way of knowing the DevicePixelRatio of the screen because that class is not a QWidget derived class. That of course was my first thought too :)

I can put this whole QObject::connect between the #ifdefs for SPOUT to reduce unecessary SLOT calls. I'll do that.

Reply:
Ok